shell.nix

```diff
@@ -25,7 +35,8 @@ ghc       = ghcOrig.override (oldArgs: {
 ###
 drvf =
```

Comment:
This is actually something that needs to be automated as almost PR adds/removes a dependency to our code base.